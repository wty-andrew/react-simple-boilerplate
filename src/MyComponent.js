import React, { Component } from 'react'

import './MyComponent.scss'

export class MyComponent extends Component {
  render() {
    return <div>MyComponent</div>
  }
}

export default MyComponent
