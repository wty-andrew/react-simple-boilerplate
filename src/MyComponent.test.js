import React from 'react'
import { render } from '@testing-library/react'

import MyComponent from './MyComponent'

describe('<MyComponent />', () => {
  it('renders correctly', () => {
    const { getByText, asFragment } = render(<MyComponent />)
    expect(getByText('MyComponent')).toBeInTheDocument()
    expect(asFragment()).toMatchSnapshot()
  })
})
