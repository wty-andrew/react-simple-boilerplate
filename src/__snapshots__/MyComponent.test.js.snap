// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyComponent /> renders correctly 1`] = `
<DocumentFragment>
  <div>
    MyComponent
  </div>
</DocumentFragment>
`;
